docs/reference/installation.md

```diff
@@ -0,0 +1,123 @@
+```
+Then, clone the toolkit:
+```bash
+git clone --depth 1 --recurse-submodules -j8 https://github.com/opendr-eu/opendr
+```
+You are then ready to install the toolkit:
+```bash
+cd opendr
+./bin/install.sh
+```
+The installation script automatically installs all the required dependencies.
+Note that you can set the inference device using the `OPENDR_DEVICE` variable.
+The toolkit defaults to using CPU.
+If you want to use GPU, please set this variable accordingly:
+```bash
+export OPENDR_DEVICE=gpu
+```
+The installation script creates a *virtualenv*, where the toolkit is installed.
+To activate OpenDR environment you can just source the `activate.sh`:
+```bash
+source ./bin/activate.sh
+```
+Then, you are ready to use the toolkit!
+
+
+If you plan to use GPU-enabled functionalities, then you are advised to install [CUDA 10.2](https://developer.nvidia.com/cuda-10.2-download-archive).
+To do so, you can follow these steps:
+```bash
+sudo apt install gcc-8 g++-8 gcc-9 g++-9
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 8 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 9 --slave /usr/bin/g++ g++ /usr/bin/g++-9
+echo "Please switch to GCC 8"
+sudo update-alternatives --config gcc
+```
+Then, you can install CUDA, along CuDNN. 
+You can also refer to this [Dockerfile](https://github.com/opendr-eu/opendr/blob/master/Dockerfile-cuda) for installation instructions.
+Note that NVIDIA 30xx GPUs may not be fully supported, due to CUDA limitations.
+
+# Installing using *pip*
+
+You can directly install OpenDR toolkit for CPU-only inference using pip.
+First, install the required dependencies:
+```bash
+export DISABLE_BCOLZ_AVX2=true
+sudo apt install python3.8-venv libfreetype6-dev git build-essential cmake python3-dev
+python3 -m venv venv
+source venv/bin/activate
+pip install Cython torch==1.7.1 wheel
+pip install git+https://github.com/cidl-auth/cocoapi@03ee5a19844e253b8365dbbf35c1e5d8ca2e7281#subdirectory=PythonAPI
+pip install git+https://github.com/cocodataset/panopticapi.git@7bb4655548f98f3fedc07bf37e9040a992b054b0
+pip install git+https://github.com/MatthewHowe/DCNv2@194f5733c667cf13e5bd478a8c5bf27573ffa98c
+```
+Then, you can install the Python API of the toolkit using pip:
+```bash
+pip install opendr-toolkit
+```
+For some systems it is necessary to disable AVX2 during `bcolz` dependency installation (`export DISABLE_BCOLZ_AVX2=true`).
+Please note that only the Python API is exposed when you install OpenDR toolkit using *pip*.
+*Note: This mode of installation is not yet available, since wheel is not yet published in PyPI.*
+
+*pip* wheels only install code that is available under the *src/opendr* folder of the toolkit.
+Tools provided in *projects* are not installed by *pip*.
+
+# Installing using *docker*
+## CPU docker
+After installing [docker](https://docs.docker.com/engine/install/ubuntu/), you can build our docker container (based on Ubuntu 20.04):
```

Comment:
Perhaps we could also mention that you should run this command in the folder where the docker is located?

Reply:
That's a good note. Probably, more detailed instructions would have also made it work on my machine.

Reply:
As I understood, the docker build should work without cloning the repo manually, which means that the user only needs a link to a Dockerfile (which is in the root folder now), so we should add the link to it here in order to make it easier to download it and start the docker.

Reply:
I think the image, once built, will be uploaded to dockerhub so that it can be run with a `docker run -it opendr-eu/opendr-toolkit:latest`. These instructions are more about building it locally

Reply:
@ad-daniel In this case, should we have instructions for two ways to use Docker, locally and via dockerhub?

Reply:
I updated the instructions. I will add the commands for pulling the docker images later today.